native/src/main/scala/sqs4s/api/lo/SendMessage.scala

```diff
@@ -0,0 +1,97 @@
+package sqs4s.api.lo
+
+import cats.effect.{Clock, Sync}
+import cats.implicits._
+import org.http4s.client.Client
+import org.http4s.scalaxml._
+import org.http4s.{Method, Request, Uri}
+import sqs4s.api.SqsSetting
+import sqs4s.api.errors.SqsError
+import sqs4s.internal.aws4.common._
+import sqs4s.internal.models.CReq
+import sqs4s.serialization.MessageEncoder
+
+import scala.concurrent.duration.Duration
+import scala.xml.{Elem, XML}
+
+case class SendMessage[F[_]: Sync: Clock, T](
+  message: T,
+  queueUrl: String,
+  attributes: Map[String, String] = Map.empty,
+  delay: Option[Duration] = None,
+  dedupId: Option[String] = None,
+  groupId: Option[String] = None
+)(implicit encoder: MessageEncoder[F, T, String, String])
+    extends Action[F, SendMessage.Result] {
+
+  def runWith(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)